.builder/actions/crt_size_check.py

```diff
@@ -9,33 +9,29 @@
 class CrtSizeCheck(Builder.Action):
     def run(self, env):
         # Maximum package size in bytes
-        max_node_size = 5_000_000
+        max_size = 5_000_000
         # Maximum dist folder size in bytes
-        max_dist_size = 50_000_000
-        # Size of current folder
-        current_folder_size = 0
-        # Total size of files in dist folder
         total_size = 0
 
-        for root, dirs, files in os.walk(os.path.join(env.project.path, 'dist/')):
-            current_folder_size = 0
+        for root, dirs, files in os.walk(os.path.join(env.project.path, 'dist/native/')):
 
             for f in files:
                 fp = os.path.join(root, f)
-                current_folder_size += os.path.getsize(fp)
+                total_size += os.path.getsize(fp)
+
+        print(f"/native files total size: {total_size} bytes")
 
+        for root, dirs, files in os.walk(os.path.join(env.project.path, 'dist/')):
+            for f in files:
                 if 'aws-crt-nodejs.node' == f:
+                    fp = os.path.join(root, f)
```

Comment:
what about the .js files in here? Those are also distributed

Reply:
Just spoke with Dengke. Going to change it to include all files in the dist folder except the ones in lib and include.

Reply:
actually, come to think of it, everything in dist/ is shipped